Assignment2/hunter097-1.b.ttl

```diff
@@ -0,0 +1,18 @@
+@base < www.oeg upm.net / person >
+@prefix : < www.oeg upm.net / person #>
+@prefix person : < www.oeg upm.net / person #>
+
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Measurement8401:hasMeasurement a rdf:Sensor029;
+	rdfs:hasTemperature:"29"^^xsd:integer;

```

Comment:
Wrong syntax still. Check other examples from other students, already accepted, and you will realise about them.

Reply:
i was checking other examples, i think the error was in the ":" of line 14